src/client/api/user.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+
+import APIS from '../constants/apis.js';
+import PATHS from '../constants/paths';
+
+function login (instance, userData) {
+  return axios
+    .post(APIS.login, { userData })
+    .then(function (response) {
+      instance.setState({ userId: response.data.insertedId[0] });
+      instance.routeChange(PATHS.trips(instance.state.userId));
+    })
+    .catch(function (error) {
+      if (error.response && error.response.status === 401) {
+        instance.routeChange(PATHS.landingPage);
+        return;
+      }
+      alert(error.message);
+    });
+}
+
+function createGuestUser (instance) {
+  return axios
+    .post(APIS.createGuestUser)
+    .then(function (response) {
+      instance.setState({ userId: response.data.insertedId[0] });
```

Comment:
Didn't we agree that the user Id will not be given to the UI? It will only be the token, as we discussed earlier

Reply:
You can see my PR to see the implementation btw, if there is an issue lmk